### ISSUE TITLE
Auto-completion window no longer shows table fields in SQL editor

### ISSUE THREAD

**Issue by Jailyard90Grad** [TablePlus/TablePlus#1435]
1. Which driver are you using and version of it (Ex: PostgreSQL 10.0):
PostgreSQL 9.6

2. Which TablePlus build number are you using (the number on the welcome screen, Ex: build 81):
261

3. The steps to reproduce this issue:
SELECT * 
from foo f 
join bar b on f.bar_id = b.id
where f.value = 3
order by b.last

In the past I would type f. or b. and the fields available in each table would appear in the auto-complete window. Now they do not. 

I used this all the time and it's a bit disconcerting not to know which table a field is referencing if we have very similarly named fields. Is there a way to enable this again? Or perhaps there's a new setting that I've missed that enables this?

Noted: If the bug is related to data, please attach an example SQL data.

**Comment by huyphams** [comment]
Hey thanks @Jailyard90Grad for the feedback, I couldn't reproduce the issue at my end. It seems that it only happens in some particular cases because I didn't change anything related to PostgreSQL recently.  I will double-check it.
<img width="671" alt="Screen Shot 2019-09-17 at 3 49 05 PM" src="https://user-images.githubusercontent.com/3654622/65070274-b2206280-d962-11e9-97f8-0eeaee24cf36.png">

**Comment by huyphams** [comment]
I can reproduce it now, I'm fixing it.

**Comment by huyphams** [comment]
The build 262 is released, please let me know if you're still seeing the issue. Thank you!

**Comment by Jailyard90Grad** [comment]
It's working again! Thanks.

**Comment by huyphams** [comment]
Just in case you still have this issue, please read this post: https://twitter.com/TablePlus/status/1174320380913311749
TL;DR download: https://tableplus.com/release/osx/tableplus_hotfix
